securebanking-openbanking-uk-rcs-sample/docker/docker-compose.yml

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright © 2020 ForgeRock AS (obst@forgerock.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+version: '3.8'
+services:
+
+  securebanking-config-server:
+    container_name: securebanking-config-server
+    hostname: securebanking-config-server
+    image: eu.gcr.io/openbanking-214714/securebanking/securebanking-config-server:latest
+    environment:
+      - SPRING_PROFILES_ACTIVE=native
+      - CONFIG_SERVER_SEARCH_LOCATIONS=file:///home/config
+    ports:
+      - 8888:8888
+    volumes:
+      - ./config:/home/config
+
+  rcs-service:
+    container_name: rcs-service
+    image: eu.gcr.io/openbanking-214714/securebanking/uk-ob-rcs:latest
+    ports:
+      - 8080:8080
+    environment:
+      - SPRING_PROFILES_ACTIVE=docker
+      - SPRING_CLOUD_CONFIG_URI=http://securebanking-config-server:8888
+    restart: on-failure
+    depends_on:
+      - securebanking-config-server
+      - wiremock
+      - rs-simulator
+    links:
+      - securebanking-config-server
+      - wiremock
+      - rs-simulator
+
+  rs-simulator:
```

Comment:
The "real" RS simulator which uses the `securebanking-openbanking-uk-rs-docker.yml` config file above.